src/base-apis.js

```diff
@@ -26,6 +29,17 @@ limitations under the License.
 const httpApi = require("./http-api");
 const utils = require("./utils");
 
+function termsUrlForService(serviceType, baseUrl) {
+    switch (serviceType) {
+        case SERVICETYPES.IS:
+            return baseUrl + httpApi.PREFIX_IDENTITY_V2;
```

Comment:
The MSC suggests this is `/GET $prefix/terms` for both ISes and IMs, but the IS here is missing the `/terms` suffix.